.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
```

Comment:
Do we even need travis is you're using gitlab ci now? Should we just remove this?

Reply:
I think it can be safely removed, yeah.